index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dr. Norman Borlaug</title>
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body id="main">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <p>The man who saved a billion lives</p>
+    <div class='landpage' id="img-div">
+            <img src="images/image1.jpg" alt="Dr. Norman Borlaug with people" id="image">
+            <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+    </div>
```

Comment:
- [ Optional] Kindly, try to be as clear as possible your comment here is not understandable.

Reply:
@algerina thanks a lot I just noticed I cannot put tags inside another markup language 😄 so that was the problem why my comments look messy thanks a lot for  notifying me